sencha-workspace/tests/index.js

```diff
@@ -0,0 +1,20 @@
+var harness = new Siesta.Harness.Browser.ExtJS();
+
+harness.configure({
+    title       : 'My Test Suite',
+
+    preload     : [
+        // version of ExtJS used by your application 
+        // (not needed if you use Sencha Cmd which builds a complete 'all-file' including Ext JS itself)
+        '../ext-6.0.1/build/ext-all.js',
+        '../ext-6.0.1/build/ext-debug.js',
```

Comment:
@aliyeysides you wouldn't load ext-all and ext-debug

`ext-all` and `ext-all-debug` have the same code, but -debug isn't compressed
`ext` and `ext-debug` have the same code, but -debug isn't compressed

the `ext-all` varieties include everything in `ext` (the core) + the entire UI framework

Plus, we're loading the production built app.js below which includes `ext` + all the Ext UI used in the app + all the app classes

The only file you should have to load here is the built app.js below. But if you were loading the unbuilt app.js, you would only load one file from /ext/build/ext-*